modules/aws/sn-volume-access/variables.tf

```diff
@@ -0,0 +1,75 @@
+variable "sn_policy_version" {
+  description = "The value of SNVersion tag"
+  default     = "3.16.1" # {{ x-release-please-version }}
+  type        = string
+}
+
+variable "region" {
+  default     = "*"
+  description = "The AWS region where your instance of StreamNative Cloud is deployed. Defaults to all regions \"*\""
+  type        = string
+}
+
+variable "streamnative_vendor_access_role_arns" {
+  default     = ["arn:aws:iam::311022431024:role/cloud-manager"]
+  description = "This role for access customer s3 bucket on control plane."
+  type        = list(string)
+}
+
+variable "additional_federated_identifiers" {
+  default     = []
+  description = "This federated identified list for access customer s3 bucket on data plane."
+  type        = list(string)
+}
+
+variable "streamnative_principal_ids" {
+  default     = []
+  description = "When set, this applies an additional check for certain StreamNative principals to futher restrict access to which services / users can access an account."
+  type        = list(string)
+}
+
+variable "source_identities" {
+  default     = []
+  description = "Place an additional constraint on source identity, disabled by default and only to be used if specified by StreamNative"
+  type        = list(any)
+}
+
+variable "source_identity_test" {
+  default     = "ForAnyValue:StringLike"
+  description = "The test to use for source identity"
+  type        = string
+}
+
```

Comment:
We may need to support passing `external_id` as an array.

Reply:
We should do [it](https://github.com/streamnative/terraform-managed-cloud/blob/decac0cdf76361eddb91011bcb98b5b8ab2d9188/modules/aws/vendor-access/variables.tf#L60) from day 1 to avoid further migration.